include/amici/model.h

```diff
@@ -763,6 +763,16 @@ namespace amici {
         
         const realtype gett(const int it, const ReturnData *rdata) const;
 
+        /**
```

Comment:
we dont need double doxygen documentation

Reply:
Why double? The one below is a different function.